### ISSUE TITLE
Replace is case insenstitive

### ISSUE THREAD

**Issue by dfindlay** [knom/vsts-regex-tasks#34]
The RegEx replace performed is case insenstitive, since it uses the PowerShell Replace operator which by default is case insensitive (https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-7.3#replacement-with-regular-expressions). This limits the usability of this task to situations where case is irrelevant only.

There is another operator "creplace" in Powershell which is cases sensitive. Would be good to at least have the option to use this to do case sensitive replacements.

**Comment by ALiwoto** [comment]
I totally agree with having the option to do case- sensitive/insensitive comparision.
